docs/source/components/ops/abs.rst

```diff
@@ -0,0 +1,66 @@
+.. include:: ../../mydefs.rst
+
+``abs``
+========
+
+Header: ``<pressio/ops.hpp>``
+
+API
+---
+
+.. code-block:: cpp
+
+  namespace pressio { namespace ops{
+
+  template <class T1, class T2>
+  void abs(T1 & destination, const T2 & source);
+
+  }} // end namespace pressio::ops
+
+Parameters
+----------
+
+* ``destination``: the object to store absolute value
+
+* ``source``: the object to calculate abs
```

Comment:
"the object whose elements are used to compute the absolute value of"

Reply:
Updated, thanks!